File: app/src/main/java/com/jakelauer/baseballtheater/SplashActivity.java
package com.jakelauer.baseballtheater;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

/**
 * Created by Jake on 1/21/2017.
 */

public class SplashActivity extends AppCompatActivity
{

	@Override
	protected void onCreate(Bundle savedInstanceState)
	{
		super.onCreate(savedInstanceState);

		Intent intent = new Intent(this, MainActivity.class);
		startActivity(intent);
		finish();
	}
}

File: app/src/main/java/com/jakelauer/baseballtheater/SplashActivity.kt
package com.jakelauer.baseballtheater

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.jakelauer.baseballtheater.experiences.nux.NuxActivity
import com.jakelauer.baseballtheater.utils.PrefUtils

/**
 * Created by Jake on 1/21/2017.
 */

class SplashActivity : AppCompatActivity()
{

	override fun onCreate(savedInstanceState: Bundle?)
	{
		super.onCreate(savedInstanceState)

		val seenNux = PrefUtils.getBoolean(this, PrefUtils.SEEN_NUX)
		val activityToStart = if (seenNux) MainActivity::class.java else NuxActivity::class.java

		val intent = Intent(this, activityToStart)
		startActivity(intent)
		finish()
	}
}